cirrus/server/Dockerfile

```diff
@@ -70,10 +70,23 @@ COPY --from=rust-builder /application-services/fml.py /application-services/fml_
 COPY --from=python-builder /usr/local/lib/python3.11/site-packages /usr/local/lib/python3.11/site-packages
 COPY --from=python-builder /usr/local/bin /usr/local/bin
 
+# Set environment variables
 ENV PYTHONPATH=$PYTHONPATH:/application-services
+ENV PORT=8001
+
+# Create an app group with GID of 10001
+RUN groupadd -g 10001 app
+
+# Create an app user with UID of 10001 and make them a member of the app group
+RUN useradd -u 10001 -g 10001 -m -s /bin/sh app
+
+RUN chown -R app:app /app
+
+# Set the user to "app" and accept configuration through environment variables
+USER app
 
-# Copy all other files into the container's working directory
-COPY . .
+# Copy all other files into the container's working directory (/app)
+COPY  . /app
 
 # Start application with Uvicorn server
-CMD ["uvicorn", "cirrus.main:app", "--host", "0.0.0.0", "--port", "8001"]
+CMD ["sh", "-c", "uvicorn cirrus.main:app --host 0.0.0.0 --port $PORT"]
```

Comment:
Why are these changes necessary to add the healthchecks?

Reply:
I have asked the same question to Brandon too, and waiting for his reply

Reply:
I would expect the healthcheck pr to only add the endpoints and tests.  If these changes are necessary for some other reason I'd pull them into a separate PR.

Reply:
I agree, so I took out the changes and this PR only contains changes related to end points, updated description as well.